protoc_plugin/lib/file_generator.dart

```diff
@@ -312,6 +314,17 @@ class FileGenerator extends ProtobufContainer {
     }
   }
 
+  void _writeWellKnownImports(IndentingWriter out) {
+    for (var m in messageGenerators) {
+      var wellKnownType = wellKnownTypeForFullName(m.fullName);
```

Comment:
Nit: this could be written:
```dart
for (final message in messageGenerators) {
  final importStatements = wellKnownTypeForFullName(message.fullName)?.extraImports ?? <String>[];
  for (String importStatement in  importStatements) {
    out.println(importStatement);
  }
}
```